src/app/app.component.ts

```diff
@@ -93,13 +122,12 @@ export class AppComponent implements OnInit {
     window.addEventListener('resize', onResize.bind(this));
   }
 
+  ngOnDestroy(): void {
+    this.oidcSecurityService.onModuleSetup.unsubscribe();
+  }
+
   logout() {
```

Comment:
endpoint service can be the delegate or representative of oidcSecurityService that glowing bear app talks to

Reply:
OK, this is going in "AuthService"